### PR TITLE
New translations inclusive CN 5.3

### PR DESCRIPTION
**Pull request by Akurosia**

### CODE REVIEW

File: ui/raidboss/data/05-shb/trial/wol-ex.js [hunk 1]
```diff
@@ -912,6 +942,52 @@ export default {
...
+        'Spectral Summoner': '幻光召唤师',
+        'Spectral Egi': '幻光召唤兽',
+        'Wyrm Of Light': '光之真龙',
+        'Spectral Bard': '幻光吟游诗人',
+        'Spectral White Mage': '幻光白魔法师',
+      },
+      'replaceText': {
+        'Absolute Flash': '绝对闪光',
+        'Absolute Holy': '绝对神圣',
+        'Absolute Stone III': '绝对垒石',
```
Comment (ShadyWhite) [inline ui/raidboss/data/05-shb/trial/wol-ex.js:963]:
```suggestion
        'Absolute Stone III': '绝对垒石',
        'Berserk': '狂暴',
```

File: ui/raidboss/data/05-shb/trial/wol-ex.js [hunk 2]
```diff
@@ -912,6 +942,52 @@ export default {
...
+        'Blade Of Shadow': '漆黑魔剑',
```
Comment (ShadyWhite) [inline ui/raidboss/data/05-shb/trial/wol-ex.js:964]:
```suggestion
        'Blade Of Shadow': '漆黑魔剑',
        'BLM/WHM': '黒魔／白魔',
```

File: ui/raidboss/data/05-shb/trial/wol-ex.js [hunk 3]
```diff
@@ -912,6 +942,52 @@ export default {
...
+        'Brimstone Earth': '狱火大地',
```
Comment (ShadyWhite) [inline ui/raidboss/data/05-shb/trial/wol-ex.js:965]:
```suggestion
        'Brimstone Earth': '狱火大地',
        '(?<!\\w)Cast(?= )': '五连',
        'Cauterize': '灼热俯冲',
```

File: ui/raidboss/data/05-shb/trial/wol-ex.js [hunk 4]
```diff
@@ -912,6 +942,52 @@ export default {
...
+        'Coruscant Saber': '光明利剑',
+        'Deep Darkside': '深度暗黑',
+        'Deluge Of Death': '死亡暴雨',
```
Comment (ShadyWhite) [inline ui/raidboss/data/05-shb/trial/wol-ex.js:968]:
```suggestion
        'Deluge Of Death': '死亡暴雨',
        'DRK/BRD': '黑骑／诗人',
        'Elddragon Dive': '远古龙炎冲',
```

File: ui/raidboss/data/05-shb/trial/wol-ex.js [hunk 5]
```diff
@@ -912,6 +942,52 @@ export default {
...
+        'Fatal Cleave': '夺命飞环',
+        'Fire': '火炎',
```
Comment (ShadyWhite) [inline ui/raidboss/data/05-shb/trial/wol-ex.js:970]:
```suggestion
        'Fire': '火',
        'Flare Breath': '核爆吐息',
```

File: ui/raidboss/data/05-shb/trial/wol-ex.js [hunk 6]
```diff
@@ -912,6 +942,52 @@ export default {
...
+        '(?<!Absolute )Holy': '神圣',
```
Comment (ShadyWhite) [inline ui/raidboss/data/05-shb/trial/wol-ex.js:971]:
```suggestion
        '(?<!Absolute )Holy': '神圣',
        'Ice': '冰',
```

File: ui/raidboss/data/05-shb/trial/wol-ex.js [hunk 7]
```diff
@@ -912,6 +942,52 @@ export default {
...
+        'Imbued Coruscance': '魔法剑技·光明利剑',
```
Comment (ShadyWhite) [inline ui/raidboss/data/05-shb/trial/wol-ex.js:972]:
```suggestion
        'Imbued Coruscance': '魔法剑技·光明利剑',
        'Imbued Fire/Ice': '魔法剑(火／冰)',
        'Imbued Holy': '魔法剣 (光)',
        'Imbued Ice/Fire': '魔法剣 (冰／火)',
        'Imbued Stone': '魔法剣  (土)',
        'Katon\\: San': '叁式火遁之术',
        '(?<! )Limit(?! Break)': '极限技',
        'Limit Break': '极限技',
        'Meteor Impact': '陨石冲击',
        '(?<= )NIN': '忍者',
```

File: ui/raidboss/data/05-shb/trial/wol-ex.js [hunk 8]
```diff
@@ -912,6 +942,52 @@ export default {
...
+        'Perfect Decimation': '完美地毁人亡',
+        'Quintuplecast': '五连咏唱',
+        'Radiant Braver': '',
```
Comment (ShadyWhite) [inline ui/raidboss/data/05-shb/trial/wol-ex.js:975]:
```suggestion
        'Radiant Braver': '光之勇猛烈斩',
```

File: ui/raidboss/data/05-shb/trial/wol-ex.js [hunk 9]
```diff
@@ -912,6 +942,52 @@ export default {
...
+        'Radiant Desperado': '光之亡命暴徒',
+        'Radiant Meteor': '光之陨石流星',
+        'Shining Wave': '光芒波动',
```
Comment (ShadyWhite) [inline ui/raidboss/data/05-shb/trial/wol-ex.js:978]:
```suggestion
        'Shining Wave': '光芒波动',
        'SMN/WAR': '召唤／战士',
```

File: ui/raidboss/data/05-shb/trial/wol-ex.js [hunk 10]
```diff
@@ -912,6 +942,52 @@ export default {
...
+        'Solemn Confiteor': '庄严悔罪',
+        'Specter Of Light': '幻光召唤',
+        '(?<!Absolute )Stone(?! Earth)': '飞石',
```
Comment (ShadyWhite) [inline ui/raidboss/data/05-shb/trial/wol-ex.js:981]:
```suggestion
        '(?<!Absolute )Stone(?! Earth)': '飞石',
        'Suiton\\: San': '叁式水遁之术',
        'Summon(?! Wyrm)': '召唤',
```

File: ui/raidboss/data/05-shb/trial/wol-ex.js [hunk 11]
```diff
@@ -912,6 +942,52 @@ export default {
         'Ultimate Crossover': 'アルティメット・クロスオーバー',
       },
     },
+    {
+      'locale': 'cn',
+      'missingTranslations': true,
```
Comment (ShadyWhite) [inline ui/raidboss/data/05-shb/trial/wol-ex.js:947]:
```suggestion
```

**Review by ShadyWhite — CHANGES_REQUESTED** [review]
Fix missingTranslations

File: ui/raidboss/data/05-shb/trial/wol.js [hunk 12]
```diff
@@ -516,6 +532,51 @@ export default {
         'Ultimate Crossover': 'アルティメット・クロスオーバー',
       },
     },
+    {
+      'locale': 'cn',
+      'missingTranslations': true,
```
Comment (ShadyWhite) [inline ui/raidboss/data/05-shb/trial/wol.js:537]:
```suggestion
```

File: ui/raidboss/data/05-shb/trial/wol.js [hunk 13]
```diff
@@ -516,6 +532,51 @@ export default {
...
+        'Spectral Black Mage': '幻光黑魔法师',
+        'Spectral Dark Knight': '幻光暗黑骑士',
+        'Spectral Egi': '幻光召唤兽',
+        'Spectral Ninja': '幻光忍者',
+        'Spectral Summoner': '幻光召唤师',
+        'Spectral Warrior': '幻光狂战士',
+        'Warrior Of Light': '光之战士',
+        'Wyrm Of Light': '光之真龙',
+      },
+      'replaceText': {
```
Comment (ShadyWhite) [inline ui/raidboss/data/05-shb/trial/wol.js:549]:
```suggestion
      'replaceText': {
      '\\(In\\)': '(月环)',
      '\\(Out\\)': '(钢铁)',
      '--active time event--': '--XJB按--',
```

File: ui/raidboss/data/05-shb/trial/wol.js [hunk 14]
```diff
@@ -516,6 +532,51 @@ export default {
...
+        'Absolute Blizzard III': '绝对冰封',
+        'Absolute Fire III': '绝对爆炎',
```
Comment (ShadyWhite) [inline ui/raidboss/data/05-shb/trial/wol.js:551]:
```suggestion
        'Absolute Fire III': '绝对爆炎',
        'Absolute Fire/Blizard': '绝对爆炎/绝对冰封',
```

File: ui/raidboss/data/05-shb/trial/wol.js [hunk 15]
```diff
@@ -516,6 +532,51 @@ export default {
...
+        'Absolute Holy': '绝对神圣',
+        'Absolute Teleport': '绝对传送',
+        'Ascendance': '生辰星位',
+        'Brimstone Earth': '狱火大地',
+        'Cauterize': '',
```
Comment (ShadyWhite) [inline ui/raidboss/data/05-shb/trial/wol.js:556]:
```suggestion
        'Cauterize': '灼热俯冲',
```

File: ui/raidboss/data/05-shb/trial/wol.js [hunk 16]
```diff
@@ -516,6 +532,51 @@ export default {
...
+        'Coruscant Saber': '光明利剑',
+        'Deluge Of Death': '死亡暴雨',
+        'Elddragon Dive': '',
```
Comment (ShadyWhite) [inline ui/raidboss/data/05-shb/trial/wol.js:559]:
```suggestion
        'Elddragon Dive': '远古龙炎冲',
```

File: ui/raidboss/data/05-shb/trial/wol.js [hunk 17]
```diff
@@ -516,6 +532,51 @@ export default {
...
+        'Flare Breath': '',
```
Comment (ShadyWhite) [inline ui/raidboss/data/05-shb/trial/wol.js:560]:
```suggestion
        'Flare Breath': '核爆吐息',
```

File: ui/raidboss/data/05-shb/trial/wol.js [hunk 18]
```diff
@@ -516,6 +532,51 @@ export default {
...
+        'Imbued Coruscance': '魔法剑技·光明利剑',
+        'Meteor Impact': '陨石冲击',
+        'Perfect Decimation': '完美地毁人亡',
+        'Radiant Braver': '',
```
Comment (ShadyWhite) [inline ui/raidboss/data/05-shb/trial/wol.js:564]:
```suggestion
        'Radiant Braver': '光之勇猛烈斩',
```

**Review by ShadyWhite — CHANGES_REQUESTED** [review]
Fix missingTranslations

File: ui/raidboss/data/05-shb/alliance/the_puppets_bunker.js [hunk 19]
```diff
@@ -1076,5 +1104,97 @@ export default {
...
+      'missingTranslations': true,
+      'replaceSync': {
+        '724P-operated superior flight unit \\(A-lpha\\)': '724P：强化型飞行装置[A-lpha]',
+        '767P-Operated Superior Flight Unit \\(B-Eta\\)': '767P：强化型飞行装置[B-eta]',
+        '772P-Operated Superior Flight Unit \\(C-Hi\\)': '772P：强化型飞行装置[C-hi]',
+        '813P-Operated Aegis Unit': '813P：装备据点防卫装置',
+        '905P-Operated Heavy Artillery Unit': '905P：装备重型陆战装置',
+        'Compound 2P': '2P：融合体',
+        'Compound Pod': '辅助机：融合体',
+        'Flight Unit': '飞行装置',
```
Comment (ShadyWhite) [inline ui/raidboss/data/05-shb/alliance/the_puppets_bunker.js:1118]:
```suggestion
        '(?<!Superior )Flight Unit': '飞行装置',
```

File: ui/raidboss/data/05-shb/alliance/the_puppets_bunker.js [hunk 20]
```diff
@@ -1076,5 +1104,97 @@ export default {
...
+        '(?<!Compound )Pod': '辅助机',
+        'Puppet 2P': '2P：分裂体',
+        'The Compound': '融合的人偶群',
+        'The elevated detritus': '残骸A',
+        'The sunken detritus': '残骸B',
+        'The launch deck': '发射甲板',
+        'Core Command': '第二司令室',
+        'The passage': '通道',
+      },
+      'replaceText': {
```
Comment (ShadyWhite) [inline ui/raidboss/data/05-shb/alliance/the_puppets_bunker.js:1129]:
```suggestion
      'replaceText': {
      '(?<=\\()In': '内',
      'Out(?=\\)|/)': '外',
      '(?<=\\(|/)Behind(?=\\))': '后',
```

File: ui/raidboss/data/05-shb/alliance/the_puppets_bunker.js [hunk 21]
```diff
@@ -1076,5 +1104,97 @@ export default {
...
+        'Maneuver: Martial Arm': '攻击：格斗机械臂',
+        'Maneuver: Missile Command': '攻击：导弹全弹发射',
+        'Maneuver: Precision Guided Missile': '攻击：高性能诱导导弹',
+        'Maneuver: Refraction Cannons': '攻击：偏向粒子炮',
+        'Maneuver: Revolving Laser': '攻击：回旋激光',
+        'Maneuver: Saturation Bombing': '攻击：全方位导弹轰炸',
+        'Maneuver: Unconventional Voltage': '攻击：集束电压',
+        'Maneuver: Volt Array': '攻击：扩散电压',
+        'Mechanical Contusion': '碎机光击',
+        'Mechanical Decapitation': '斩机光击',
```
Comment (ShadyWhite) [inline ui/raidboss/data/05-shb/alliance/the_puppets_bunker.js:1177]:
```suggestion
        'Mechanical Decapitation(?!/)': '斩机光击',
```

File: ui/raidboss/data/05-shb/alliance/the_puppets_bunker.js [hunk 22]
```diff
@@ -1076,5 +1104,97 @@ export default {
...
+        'Mechanical Dissection': '断机光击',
```
Comment (ShadyWhite) [inline ui/raidboss/data/05-shb/alliance/the_puppets_bunker.js:1178]:
```suggestion
        'Mechanical Decapitation/Dissection': '斩机光击/断机光击',
        'Mechanical Dissection(?!/)': '断机光击',
        'Mechanical Dissection/Decapitation': '断机光击/斩机光击',
```

File: ui/raidboss/data/05-shb/alliance/the_puppets_bunker.js [hunk 23]
```diff
@@ -1076,5 +1104,97 @@ export default {
...
+        'Operation: Pod Program': '通信：辅助机程序',
+        'Operation: Synthesize Compound': '通信：爆炸性化合物',
+        'Prime Blade': '斩机击：填充',
+        'R010: Laser': 'R010:激光',
+        'R011: Laser': 'R011：激光',
+        'R012: Laser': 'R012：激光',
+        'R030: Hammer': 'R030:重锤',
+        'Relentless Spiral': '涡状光线奔涌',
+        'Reproduce': '分裂体生成',
+        '(?<!Formation: )Sharp Turn': '回转斩击',
```
Comment (ShadyWhite) [inline ui/raidboss/data/05-shb/alliance/the_puppets_bunker.js:1192]:
```suggestion
        '(?<!Formation: )Sharp Turn': '回转斩击',
        '(?<!Formation: )Sliding Swipe': '冲锋斩击',
```

File: ui/raidboss/data/05-shb/alliance/the_puppets_bunker.js [hunk 24]
```diff
@@ -1076,5 +1104,97 @@ export default {
...
+        'Support: Pod': '支援：辅助机射出',
+        'Surface Missile Impact': '对地导弹命中',
+        'Three Parts Disdain': '三连冲击斩',
```
Comment (ShadyWhite) [inline ui/raidboss/data/05-shb/alliance/the_puppets_bunker.js:1195]:
```suggestion
        'Three Parts Disdain': '三连冲击斩',
        'Three Parts Resolve': '三连冲击斩 结束',
```

File: ui/raidboss/data/05-shb/alliance/the_puppets_bunker.js [hunk 25]
```diff
@@ -1076,5 +1104,97 @@ export default {
         'Upper Laser': '上部レーザー',
       },
     },
+    {
+      'locale': 'cn',
+      'missingTranslations': true,
```
Comment (ShadyWhite) [inline ui/raidboss/data/05-shb/alliance/the_puppets_bunker.js:1109]:
```suggestion
```

**Review by ShadyWhite — CHANGES_REQUESTED** [review]
Fix missingTranslations

**Comment by Akurosia** [comment]
@ShadyWhite there are still conflicts and missing translations.
Maybe do the rest after this one landet?

File: ui/raidboss/data/02-arr/trial/titan-ex.js [hunk 26]
```diff
@@ -67,7 +67,7 @@ export default {
       netRegex: NetRegexes.ability({ source: 'Titan', id: '285', capture: false }),
       netRegexDe: NetRegexes.ability({ source: 'Titan', id: '285', capture: false }),
       netRegexFr: NetRegexes.ability({ source: 'Titan', id: '285', capture: false }),
-      netRegexJa: NetRegexes.ability({ source: 'Titan', id: '285', capture: false }),
+      netRegexJa: NetRegexes.ability({ source: 'タイタン', id: '285', capture: false }),
```
Comment (quisquous) [inline ui/raidboss/data/02-arr/trial/titan-ex.js:70]: Oops, thank you!

**Pull request merged**